.github/pyinstaller/pyinstaller.spec

```diff
@@ -1,10 +1,40 @@
 # -*- mode: python -*-
 # Copyright (C) 2020 Mandiant, Inc. All Rights Reserved.
-import os.path
-import subprocess
+import sys
+import logging
 
 import wcwidth
 
+from pathlib import Path
+
+logger = logging.getLogger(__name__)
+
+def generate_rule_cache(capa_dir: Path):
+    import capa
+    import capa.rules
+    import capa.rules.cache
+
+    rules_dir = capa_dir / 'rules'
+    cache_dir = capa_dir / 'cache'
```

Comment:
My only concern here is that we are hardcoding the `rules` directory in the PyInstaller build process. What happens if users don't necessarily have rules in the capa root dir as a submodule`./rules/` (like I do)?

Reply:
good point, let's ensure the directory exists and otherwise fail with a warning/explanation

Reply:
If that were the case, do you recommend we set an argument (which defaults to `./rules`) that gives users the option to specify a custom dir. I tried to use `sys.argv` in `pyinstaller.spec` but coudn't get it to work.

If this is not possible, we can set an env variable `CAPA_RULES_DIR` and indicate to users to set when the warning is raised.

Reply:
let's keep it simple since I suspect few people build capa executables manually and if they do, they can hopefully figure it out given the warning